lib/chat_list.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+
+import 'chat_message.dart';
+
+class ChatList extends StatelessWidget {
+  const ChatList({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return StreamBuilder<QuerySnapshot>(
+      stream: FirebaseFirestore.instance
```

Comment:
It might not be an issue for this app for the moment but still it's not a good practice. I'm not sure if we want to introduce potentially problematic code to people.

Reply:
Just do this:
<img width="580" alt="image" src="https://github.com/fluttermtl/flutter_chat/assets/1233094/1177d9b8-8b75-45d7-b588-1074c6d3c0d8">

This should be good enough.

Reply:
I don't understand. THis doesn't fix the problem. It has to be another widget.